.github/workflows/ubuntu.yml

```diff
@@ -29,10 +29,11 @@ jobs:
       CXX: ${{ matrix.compiler-version.cxx }}
       CPLEX_URL: ${{ secrets.CPLEX129_LINUX_URL }}
       SOPLEX_URL: ${{ secrets.SOPLEX311_URL }}
-      DOWNWARD_CPLEX_ROOT: /home/runner/work/downward/lib/ibm/ILOG/CPLEX_Studio129/cplex
-      DOWNWARD_SOPLEX_ROOT: /home/runner/work/downward/lib/soplex-3.1.1
-      DOWNWARD_COIN_ROOT: /home/runner/work/downward/lib/coin
+      DOWNWARD_CPLEX_ROOT: /home/runner/lib/ibm/ILOG/CPLEX_Studio129/cplex
+      DOWNWARD_SOPLEX_ROOT: /home/runner/lib/soplex-3.1.1
+      DOWNWARD_COIN_ROOT: /home/runner/lib/coin
     steps:
+      # Check out repo at $GITHUB_WORKSPACE (= /home/runner/work/my-repo/my-repo).
```

Comment:
This is interesting to know, but how does the compilation of the planner below work without switching to this directory? 

Wouldn't it make sense to have this path more explicitly, i.e., checkout to downward in home (= /home/runner) so that we have one directory for the planner and one for the libraries?

Reply:
The checkout path is configurable, but it must lie under $GITHUB_WORKSPACE.